### PR TITLE
[TIMOB-11278] iOS: Add Ti.UI.WebView.ignoreSslError (Parity)

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-11278

### CODE REVIEW

File: iphone/Classes/TiUIWebView.h [hunk 1]
```diff
@@ -9,21 +9,23 @@
...
-	//TODO: make more elegant
-	BOOL ignoreNextRequest;
-	id reloadData;
+    BOOL ignoreNextRequest;
+    BOOL ignoreSslError;
+    BOOL isAuthenticated;
+    id reloadData;
     id reloadDataProperties;
-	SEL reloadMethod;
+    SEL reloadMethod;
```
Comment (hansemannn) [inline iphone/Classes/TiUIWebView.h]: Notes 1/2: I moved all properties to the updated indentation, so we have it easier in the future to add more properties without indentation issues.

File: iphone/Classes/TiUIWebView.m [hunk 2]
```diff
@@ -707,24 +719,25 @@ - (BOOL)webView:(UIWebView *)webView shouldStartLoadWithRequest:(NSURLRequest *)
...
+			[self loadURLRequest:newRequest];
+			[newRequest release];
+
+			return NO;
 		}
-		[self loadURLRequest:newRequest];
-		[newRequest release];
-        
-		return NO;
 	}
```
Comment (hansemannn) [inline iphone/Classes/TiUIWebView.m]: Note 2/2: I moved the `requestHeaders` logic into the `newHeaders` scope, so it doesn't affect requests without custom request headers. This fixes a possible glitch in TIMOB-24329

**Review by vijaysingh-axway — COMMENTED** [review]
insecureConnection = [[[NSURLConnection alloc] initWithRequest:request delegate:self] retain];

In above line retain will cause leak. Any reason for retaining after alloc ?

**Review by vijaysingh-axway — COMMENTED** [review]
Functionality is working fine.

**Review by vijaysingh-axway — APPROVED** [review]
FT and CR passed.

**Review by longton95 — APPROVED** [review]
FR passed.
verified that iOS devices are now able to ignore invalid SSL certificate.
Ran the test-case from the ticket and a modified version.

ENV
```
SDK Version : Local 6.1.0 & 6.0.2
Mac OS Version : 10.12.2
Xcode Version : Xcode 8.2 Build version 8C38
Appc CLI : 6.1.0
Appc NPM : 4.2.9-1
Node : v4.6.0
Device: iPhone 6 (10.2.1) iPhone 5 (9.3.5)
```

**Pull request merged**